.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint Code Base
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
+
+jobs:
+  build:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Lint Code Base
+        uses: docker://github/super-linter:v2.1.1
```

Comment:
could you check we are using the same bash ruleset which we already used in enterprise2?

Reply:
As far as I know `shellcheck` uses `#shellcheck` inline comments and `.shellcheckrc` file for single place configuration. We don't have such file in enterprise2, so I guess it falls back to inline directives and default configuration